src/js/blight/BlightContainer.jsx

```diff
@@ -1,22 +1,42 @@
 import React, { Component } from "react";
+import _ from "lodash";
+
+import blightCards from "./blight-cards.json";
 
 export default class BlightContainer extends Component {
   constructor(props) {
     super(props);
     this.totalBlight = this.props.players * 2;
 
-    this.state = { numBlight: this.totalBlight };
+    this.state = { 
+      numBlight: this.totalBlight,
+      blightCard: {},
+      endGameTrigger: false
+    };
 
     this.handleAddBlight = this.handleAddBlight.bind(this);
     this.handleRemoveBlight = this.handleRemoveBlight.bind(this);
 
   }
+
   handleAddBlight(e) {
     this.setState({ numBlight: this.state.numBlight + 1 });
   }
   handleRemoveBlight(e) {
-    this.setState({ numBlight: this.state.numBlight - 1 });
+    const newBlight = this.state.numBlight - 1;
+    this.setState({ numBlight: newBlight});
```

Comment:
like I said before, let's call `setState` only once here. 
Prepare the state object base on the condition, and at the end of the function, call `setState` with that object as the input.